src/components/Input/index.js

```diff
@@ -80,7 +86,7 @@ const CustomInput = ({ textarea, name, label, placeholder, handleChange }) => {
           placeholder={placeholderText}
           id={name}
           name={name}
-          onChange={handleChange}
+          onChange={onCLick}
```

Comment:
All is great just this one thing why this is called `onClick` as its a function that runs on change not click or am I missing something here?